onnxmltools/convert/sklearn/operator_converters/CalibratedClassifierCV.py

```diff
@@ -0,0 +1,169 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._apply_operation import apply_add, apply_cast, apply_div, apply_exp, apply_mul, apply_reshape
+from ...common._registration import get_converter, register_converter
+from .._parse import sklearn_operator_name_map
+import numpy as np
+from sklearn.naive_bayes import BernoulliNB
+from sklearn.naive_bayes import MultinomialNB
+
+
+class Oper:
+    def __init__(self, model, inputs, op_type):
+        self.raw_operator = model
+        self.inputs = inputs
+        self.type = op_type
+
+def convert_sklearn_calibrated_classifier_cv(scope, operator, container, flag=False, model=None):
+    
+    if flag:
+        supported_classifier_list = [BernoulliNB, MultinomialNB]
```

Comment:
The overall procedure looks reasonable but it'd better if they happen in the parsing stage.